### PR TITLE
Feature/coastal app

### PR DESCRIPTION
**Pull request by pvelissariou1**
Added RT configurations for SCHISM+PAHM coupled (internally and externally) cases.

### CODE REVIEW

**Comment by pvelissariou1** [comment]
@uturuncoglu Ufuk please merge.

File: tests/fv3_conf/coastal_hsofs_pahm2schism_run.IN [hunk 1]
```diff
@@ -0,0 +1,13 @@
+mkdir RESTART
```
Comment (uturuncoglu) [inline tests/fv3_conf/coastal_hsofs_pahm2schism_run.IN:1]: @pvelissariou1 Why you have multiple copy script for PAHM related configuration. I think we could manage it to have single file. As we discussed in our last call, I think, we need to separate files related with SCHSIM vs. PAHM. Files that are related with SCHSIM and needs to be in the SCHSIM specific script and RT input directory. So, for example, the files under /work/noaa/nosofs/pvelissa/RT_takis/NEMSfv3gfs/input-data-20221101/SCHISM/Sandy can be move to SCHSIM directory and if there is a configuration file specific to PAHM component that could be handled by the `fv3_conf/coastal_pahm_run.IN` (we don't have one but we could create).

**Review by uturuncoglu — COMMENTED** [review]
we could merge it, after fixing the issue in the copy scripts. Let me know what you think and if you need help to  rearrange the input scripts.

**Comment by pvelissariou1** [comment]
The SCHISM configuration on hsofs domain has extra files not present in the Shin. cases. Any idea how to combine these configs?

**Comment by uturuncoglu** [comment]
@pvelissariou1 please let me know which configuration file is for PAHM. I could fix the PR as it is and then fix it. Let me know your preference.

**Pull request merged**